testing/perl-crypt-des_ede3/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Automatically generated by apkbuild-cpan, template 2
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+pkgname=perl-crypt-des_ede3
+_pkgreal=Crypt-DES_EDE3
+pkgver=0.01
+pkgrel=0
+pkgdesc="Perl module for Crypt-DES_EDE3"
+url="https://metacpan.org/release/Crypt-DES_EDE3/"
+arch="noarch"
+license="GPL PerlArtistic"
```

Comment:
GPL-1.0-or-later Artistic-1.0-Perl

Reply:
Fixed and pushed to abuild/apkbuild-cpan.in